Source/Parallelization/WarpXComm_K.H

```diff
@@ -285,6 +285,94 @@ void warpx_interp (int j, int k, int l,
     // Final result
     arr_aux(j,k,l) = tmp + (fine - coarse);
 }
```

Comment:
This is needed by the fix in the `WarpXComm.cpp`, we need a function that interpolates between two `MultiFabs`, instead of existing functions that use 3 `MultiFabs`.

Reply:
@EZoni will check if this is functionality already exists somewhere else in WarpX.

Reply:
It looks to me like we might be able to use the `Interp` function here:
https://github.com/ECP-WarpX/WarpX/blob/b58343a9cf31ad2e2e0c47b2c5f0b15cc122a961/Source/ablastr/coarsen/sample.H#L30-L58